src/main/sql/user_detail.sql

```diff
@@ -14,9 +14,9 @@ CREATE TABLE user_detail
   
   --Constraints
   CONSTRAINT active_is_yes_or_no
-  CHECK active IN ('Y','N')
+  CHECK (active IN ('Y','N')),
 
   CONSTRAINT admin_is_yes_or_no
-  CHECK admin IN ('Y','N')
+  CHECK (admin IN ('Y','N'))
```

Comment:
Is this just for formatting or does PostGres need slightly different syntax for SQL?

Reply:
It didn't like it without the brackets. Nor does DB2 to be fair. Maybe H2 was more relaxed.